src/dirs.proj

```diff
@@ -39,4 +39,10 @@
     </TraversalBuildDependsOn>
   </PropertyGroup>
 
+  <!-- Generate the shared library for running ILC tests for configuration Release-x64 -->
```

Comment:
Any reason you put this here as opposed to at the bottom of src.builds like the shared framework generation stuff? https://github.com/dotnet/corefx/blob/master/src/src.builds#L15